src/block.tsx

```diff
@@ -88,51 +98,58 @@ export const Block: React.FC<Block> = props => {
-                />
-              )}
-              <div
-                className={classNames(
-                  "notion-page",
-                  !page_cover && "notion-page-offset",
-                  page_full_width && "notion-full-width",
-                  page_small_text && "notion-small-text"
-                )}
-              >
-                {page_icon && (
-                  <PageIcon
-                    className={
-                      page_cover ? "notion-page-icon-offset" : undefined
-                    }
-                    block={block}
-                    big
-                  />
-                )}
-                <div className="notion-title">
-                  {renderChildText(blockValue.properties.title)}
+            <div className="notion notion-app">
+              <div className="notion-cursor-listener">
+                <div className="notion-frame">
+                  <PageHeader blockMap={blockMap} mapPageUrl={mapPageUrl} />
+
+                  <div className="notion-scroller">
+                    {page_cover && (
+                      <img
+                        src={toNotionImageUrl(page_cover)}
+                        alt={getTextContent(blockValue.properties.title)}
+                        className="notion-page-cover"
+                        style={{
+                          objectPosition: `center ${coverPosition}%`
+                        }}
+                      />
+                    )}
+                    <main
+                      className={classNames(
+                        "notion-page",
+                        !page_cover && "notion-page-offset",
+                        page_full_width && "notion-full-width",
+                        page_small_text && "notion-small-text"
+                      )}
+                    >
+                      {page_icon && (
+                        <PageIcon
+                          className={
+                            page_cover ? "notion-page-icon-offset" : undefined
+                          }
+                          block={block}
+                          big
+                        />
+                      )}
+
+                      <div className="notion-title">
+                        {renderChildText(blockValue.properties.title)}
+                      </div>
+
+                      {children}
+                    </main>
+                  </div>
                 </div>
-                {children}
               </div>
             </div>
           );
         } else {
-          return <div className="notion">{children}</div>;
+          return <main className="notion">{children}</main>;
         }
       } else {
         if (!blockValue.properties) return null;
         return (
-          <a
-            className="notion-page-link"
-            href={props.mapPageUrl?.(blockValue.id) || `/${blockValue.id}`}
-          >
+          <a className="notion-page-link" href={mapPageUrl(blockValue.id)}>
```

Comment:
`mapPageUrl` now always has a default value which is cleaner than having the same fallback logic in several places.